pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Inclusive Language style guide
+description: This style guide outlines the guideline for using inclusive languages in documentation.
+---
+
+# Inclusive Language
+
+While writing documentation we are indirectly reaching out to thousands of people all around the globe. 
+Thus, we must ensure that our diverse audience can connect with our information. 
+
+### 1. Culturally Inclusive Language
+
+Some phrases and words are popularly used only in specific regions of the world, thus our audience from other parts 
+of the world might need help to relate to the same. Hence, we should try to avoid using such phrases in our documentation. 
+Some examples of Region Specific Language are -
+- “It’s not rocket science” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+- Phrases written in regional languages instead of English.
+- “It’s not a piece of cake” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+
+### 2. Gender Neutral Language - 
+
+Some phrases and words are targeting men or men/women specifically. This might result in other gender groups feeling 
+left out while reading our documentation. 
+Some examples of Gender Neutral Language are -
+- Using “they/them” instead of “his/him” or “she/her”.
+- Using “Hello everyone” instead of “Hello guys”.
+- Using “Chairperson” instead of “Chairman” or “Chairwoman”.
+
+### 3. People First Language - 
+
+The placement of the word “People” matters a lot when it comes to writing documentation. 
+Using the word “People” in the beginning makes it more pleasant as the problem is not displayed in the beginning.
+Some examples of Gender Neutral Language are -
+- Using “People with Mental Instability” instead of “Mentally Unstable People”.
+- Using “People with Audio Impairment” instead of “Deaf People”.
+
+### 4. Slang Free Language -
+
+While framing the documentation, we must ensure that we are not including any vulgar language even if those are 
+included indirectly in our work. Often we don’t realize that slang words are included within phrases which have 
+don’t have collaborative slang meanings. 
+Some examples include -
+- Using “archaic” or “stupid” instead of “moron”.
+- Avoid using words like “bloody hell” and “damn”.
+
+### 5. Ageism Free Language -  
+
+While constructing the documentation, we need to make sure to avoid words or phrases that might emphasize more 
+on a specific age group that can make the other age groups feel left out. 
+Some examples include -
+- Using “lively” instead of “young”.
+- Not to mention the exact age like “60 years” old.
+
+### 6. Knowledge Assumption Free Language - 
+
+While building the documentation, we should always assure that we don’t presume the knowledge of the readers. 
+This is because if we assume that our readers are highly skilled in front-end development, 
```

Comment:
```suggestion
This is because if we assume that our readers are highly skilled or have advanced experience, 
```

FE development isn't the only area relevant to our work, so let's fix this sentence too :)